scripts/cron/cron_run_jobs.php

```diff
@@ -42,7 +42,21 @@
 	exit(-1);
 }
 
-require_once $path."../../htdocs/master.inc.php";
+// master.inc.php should be in htdocs or in root folder in most cases
+$masterpath = ( false !== file_exists($path."../../htdocs/master.inc.php") ) ? $path."../../htdocs" : $path."../.." ;
+
+// if master.inc.php is not in htdocs or root, try to find it in root subfolder (incase of renaming htdocs)
+if ( false === file_exists($masterpath."/master.inc.php") ){
+	foreach (scandir($path."../../") as $key => $value) {
+		// test only dorectories but not project default folders
+		if ( is_dir($path."../../".$value) && !in_array($value, array( ".", "..", ".github", ".tx", "build", "dev", "doc", "documents", "scripts", "test" )) && file_exists($path."../../".$value."/master.inc.php") ){
+			$masterpath = $path."../../".$value;
+			break;
+		}
+	}
+}
+
+require_once $$masterpath."/master.inc.php";
```

Comment:
$$ masterpath

Reply:
Sorry.
It's fixed.

Reply:
Why did you change name of official path ?
Changing htdocs into xxx is same than change any dir into another name. It will break a lot of features.

Reply:
I didn't change it for my insrallation and I will not. I just tried just to bring a solution for an issue that was still open and not marked as "will not be fixed". 

Meanwhile, i must say that some provider use different version of dolibarr. For example, the version provided by infomaniak (swiss hosting provider) doesn't use htodcs: all the folder of htdocs par in the root folder .

If the version of infomaniak is more unsecure and should not be proposed, this version is very used.
The renaming of htdocs by @git-putz is probably donne to increase security.

Cheers

Reply:
How did the Dolibarr was installed ? By using an autoinstaller package, which one ? a git clone ? manually unzip ?

Reply:
On click:

https://www.infomaniak.com/fr/creer-un-site/cms/hebergement-dolibarr

Reply:
![IMG_20210219_160449](https://user-images.githubusercontent.com/8067223/108521782-60ed6e80-72cc-11eb-82b8-a6ace16230f1.jpg)